doc/PEDSnet_DQA_Documentation.md

```diff
@@ -43,19 +43,37 @@ Execute [DDL_RxNorm_Tables.sql](../Level1/library/DDL_RxNorm_Tables.sql)
 
 ### Apply DQA Checks
 
-``` R
+
+```R
 setwd("/path/to/repo/")
-source('Run_DQA.R')
+source("Run_DQA.R")
+```
+
+
+#### Run All Level 1 DQA Checks
+``` R
 runDQA(level=1)
 ```
 
-###### Level 2 For Postgres Users
+#### Run Single Level 1 DQA Check
```

Comment:
O, I put them together in case a site needed to rerun just a single report after one failed or something or after they fixed an issue. Wasn't sure if they would ever do that or we should only have it for testing purposes